### DISCUSSION TITLE
How to uninstall WSA?

### DISCUSSION THREAD

**Discussion by VsUK** [MustardChef/WSABuilds#462] [General]
I launched the bat from the folder I extracted on my desktop within WSA_2311.40000.5.0_x64_Release-Nightly-GApps-13.0-NoAmazon & now I cant move the folder as its a running service from that folder. How can I uninstall & put the folder somewhere more practical to install like my C Drive where all my programs are running from? 

I cant see any uninstall option nor is it located in uninstall programs list.

**Comment by MustardChef** [comment]
@VsUK You can move the WSA folder without uninstalling WSA. Just:

1. Make sure that WSA is not running.
2. Move the folder to your desired location.
3. Run ``Run.bat`` and run an Android app to check if WSA is working correctly.

**Reply by VsUK** [reply]
I figured it out. Couldn't move it after stopping services because when I started them again, play store or installed apps wouldn't launch. But uninstalled via apps in win11 & then moved, installed & all working.
